src/rqt_py_trees/dotcode_behaviour.py

```diff
@@ -130,6 +138,10 @@ def type_to_string(self, behaviour_type):
             return 'Sequence'
         elif behaviour_type == py_trees_msgs.Behaviour.SELECTOR:
             return 'Selector'
+        elif behaviour_type == py_trees_msgs.Behaviour.PARALLEL:
+            return 'Parallel'
+        elif behaviour_type == py_trees_msgs.Behaviour.CHOOSER:
```

Comment:
We could probably stringify the name here - python magic. Not critical though - we should have a non-increasing finite set to deal with.